README.md

```diff
@@ -1 +1,364 @@
+In the root of the 'liberty-bikes-ai' repo, run `./gradlew start` to start up your AI server.
+Next, open the server logs by going to `build/wlp/usr/servers/ai-service/logs/messages.log`. You will see content similar to this:
+
+```
+[AUDIT   ] CWWKE0001I: The server ai-service has been launched.
+[AUDIT   ] CWWKZ0058I: Monitoring dropins for applications.
+[AUDIT   ] CWWKT0016I: Web application available (default_host): http://localhost:8083/liberty-bikes-ai/
+[AUDIT   ] CWWKZ0001I: Application liberty-bikes-ai started in 0.462 seconds.
+[AUDIT   ] CWWKF0012I: The server installed the following features: [cdi-2.0, jaxrs-2.1, jaxrsClient-2.1, jndi-1.0, jsonb-1.0, jsonp-1.1, mpConfig-1.3, mpRestClient-1.3, servlet-4.0, websocket-1.1].
+[AUDIT   ] CWWKF0011I: The ai-service server is ready to run a smarter planet. The ai-service server started in 1.385 seconds.
+```
+Leave this log file open for duration of this lab -- you will be referring to it frequently to see the output of your application.
+
+
+## Hot code updates with OpenLiberty
+A very nice feature of OpenLiberty is that for local development you don't need to restart your server or even redeploy your app between code changes. If you are using an IDE that compiles the app for you, the IDE recompiling the app will be detected by Liberty and it will automatically restart the app with the new code.
+To test this out, go to the `org.libertybikes.ai.service.StartupProcedure` class and make any code change you like (as long as it compiles).
+Once you save the code change and your IDE recompiles the app, refresh your messages.log file and you will see the app updated automatically in a few milliseconds:
+
+```
+[AUDIT   ] CWWKT0017I: Web application removed (default_host): http://localhost:8083/liberty-bikes-ai/
+[AUDIT   ] CWWKZ0009I: The application liberty-bikes-ai has stopped successfully.
+[AUDIT   ] CWWKT0016I: Web application available (default_host): http://localhost:8083/liberty-bikes-ai/
+[AUDIT   ] CWWKZ0003I: The application liberty-bikes-ai updated in 0.152 seconds.
+```
+
+If you are not using an IDE that can recompile the app for you, you can manually trigger an application rebuild by running the command:
+
+```
+./gradlew war
+```
+
+## Starting the registration procedure at application startup time
+When we start the server with our AI bot application, we don't want to have to manually initiate the registration procedure with the core services. Lets write a web listener that will automatically be invoked when the application is started. Update the `StartupProcedure` class to be the following:
+
+```java
+@WebListener
+public class StartupProcedure implements ServletContextListener {
+
+    @PostConstruct
+    public void joinFirstRound() {
+        System.out.println("Hello world!");
+    }
+}
+```
+When you save this code change, the app should restart again and you should see the "Hello world!" message in your messages.log
+
+## IF you get stuck
```

Comment:
Caps?

Reply:
yea, that's intentional. Some people were going through these steps as part of the regular procedure even though they weren't stuck